dev/com.ibm.ws.transport.http2_fat/test-applications/H2FATDriver.war/src/http2/test/driver/war/servlets/H2FATDriverServlet.java

```diff
@@ -3286,9 +3587,23 @@ public void testGoAwayFrameInvalidErrorCode(HttpServletRequest request,
         CountDownLatch blockUntilConnectionIsDone = new CountDownLatch(1);
         Http2Client h2Client = getDefaultH2Client(request, response, blockUntilConnectionIsDone);
 
-        int NO_ERROR = 0x0;
-        FrameGoAway errorFrame = new FrameGoAway(0, null, NO_ERROR, 1, false);
-        h2Client.addExpectedFrame(errorFrame);
+        if (USING_NETTY) {
+            // Netty does not expect an error code back so just look for the last data frame
```

Comment:
Clean this up if no longer necessary

Reply:
@volosied

Reply:
Same comments as before.

Reply:
Link to same issue above

Reply:
https://github.com/OpenLiberty/open-liberty/issues/32386

Reply:
Actually removing in : https://github.com/OpenLiberty/open-liberty/pull/32397